### PR TITLE
[Feature] Get Requests Item List with Pagination

### PR DESCRIPTION
**Pull request by iqbal167**
# Overview
- Add Pagination Helper
- Create api getAllRequests
- Example:
Screen Shot 2022-09-28 at 15 22 38

## Todo
- Set Keycloack Middleware

## Evidence
- Title: feat: get request item list
- Project: Digiteam Inventaris Platform
- Participants: @iqbal167 @ayocodingit @sandisunandar99  @fajarhikmal214

### CODE REVIEW

File: src/models/requests/requests.service.ts [hunk 1]
```diff
@@ -19,4 +23,19 @@ export class RequestsService {
...
+    const { page = 1, limit = 10 } = getRequestsPaginateDto;
+    const result = await this.repo.fetchAll(page, limit);
+
+    const data = result.map((requests) => mapEntitytoInterface(requests));
+    const meta = metaPagination(data, page, limit);
+
+    const apiResponse: ApiResponse = {
+      data: data,
+      meta: meta,
+    };
```
Comment (fajarhikmal214) [inline src/models/requests/requests.service.ts:37]: Ini di persingkat jadi `... = { data, meta } ` kang ?

**Review by fajarhikmal214 — APPROVED** [review]
LGTM. Keren Kang 😄

**Comment by iqbal167** [comment]
> LGTM. Keren Kang 😄
hatur nuhun @fajarhikmal214 😄

**Review by sandisunandar99 — APPROVED** [review]
🚀

**Review by ayocodingit — APPROVED** [review]
Lgtm

File: src/models/requests/requests.service.ts (hunk 1 above)
Comment (iqbal167) [reply src/models/requests/requests.service.ts]: ok kang @fajarhikmal214 sudah di udapate ya, nuhunn

**Pull request merged**